scripts/update-integrations.mjs

```diff
@@ -139,6 +139,19 @@ async function unsafeUpdateAllIntegrations() {
 				const details = await fetchWithOverrides(data.name, false);
 				if (!details) return;
 
+				// check if the homepageurl is valid
+				// if not, replace it by the link to the package on npm
+				try {
+					const response = await fetch(details.homepageUrl, { method: 'HEAD'})
+					if (response.status === 404) {
+						details.homepageUrl = `https://www.npmjs.com/package/${data.name}`
+					}
+				} catch (error) {
+					// such an error may occur when the hostname is unknown
+					// we consider it ok at this may come from a down server
+					console.info(`Package ${details.name}: Cannot access ${details.homepageUrl} (not fixed)`)
```

Comment:
I'd expect this to happen more often than 404s, when a website is shut down or a domain no longer resolves. 

For the user experience, it is equally detrimental to see a 404 page or a browser error page. So I think this case should also point to NPM.

If the service is just down temporarily it will be updated back in the following weekly run of the script.